manifests/server/config.pp

```diff
@@ -49,40 +49,40 @@
         user        => $user,
         auth_method => 'ident',
         auth_option => $local_auth_option,
-        order       => '001',
+        order       => 1,
```

Comment:
This is actually working as a string on the Concat module.  `2` would be sorted after after `10`.